extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -360,6 +379,51 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     });
   }
 
+  /**
+   * Handles Fluid-related messages dispatched from SafeFrame.
+   * @param {!Event} event
+   */
+  receiveMessageForFluid(event) {
+    if (event.origin != 'https://tpc.googlesyndication.com') {
+      return;
+    }
+    const data = tryParseJson(getData(event));
+    if (!data || data['s'] != 'creative_geometry_update' ||
+        data['sentinel'] != this.sentinel) {
```

Comment:
Consider dev().warn specific to sentinel disagreement as its unexpected.  Suggest:

const data = tryParseJson(...);
if (event.origin... || !data || data['s'] != 'cre...]) { return; }
user().assert(TAG, data['sentinel'] == this.sentinel, 'some nice error message');

Reply:
Done.